### ISSUE TITLE
Incompatibility with Camera Dock and Lib Wrapper

### ISSUE THREAD

**Issue by MythicalRollers** [patrickporto/avqol#3]
Since updating to beta 4, my world freezes upon login. 
Console shows:
foundry.js:746 Uncaught (in promise) TypeError: undefined. Cannot set property message of  which has only a getter
[Detected 1 package: lib-wrapper]
    at Hooks.onError (foundry.js:746:30)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at 🎁Hooks.onError#lib-wrapper (listeners.js:138:11)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:187:52)
    at foundry.js:5069:13
    
Also under messages:
Foundry VTT | Rendering Camera views
this number just increases until my world freezes. 14899 and counting.

**Comment by patrickporto** [comment]
@MythicalRollers do you use livekit or built-in A/V client?

**Comment by MythicalRollers** [comment]
LiveKit using Bekit's patreon server.
I also use Therippers camera dock module, but the three modules worked well in Beta-3.

**Comment by MythicalRollers** [comment]
I just caught up to Foundry's releases channel and saw you sent some updates out. I'll test the latest one tonight, and let you know if my bug is resolved with your updates.

**Comment by MythicalRollers** [comment]
Latest release same issue.
Uncaught (in promise) TypeError: undefined. Cannot set property message of  which has only a getter
[Detected 1 package: lib-wrapper]
    at Hooks.onError (foundry.js:746:30)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at 🎁Hooks.onError#lib-wrapper (listeners.js:138:11)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:187:52)
    at foundry.js:5069:13
    
Foundry VTT | Rendering CameraViews 25000 and climbing
![image](https://user-images.githubusercontent.com/106455111/221726142-800ded15-85d8-48bb-bac2-b6bea57ae832.png)

**Comment by MythicalRollers** [comment]
In a test world with only LiveKit, Camera Dock and A/V QoL I get the same libWrapper error, however my world doesn't freeze:
foundry.js:746 Uncaught (in promise) TypeError: undefined. Cannot set property message of  which has only a getter
[Detected 1 package: lib-wrapper]
    at Hooks.onError (foundry.js:746:30)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at 🎁Hooks.onError#lib-wrapper (listeners.js:138:11)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:187:52)
    at foundry.js:5069:13

**Comment by MythicalRollers** [comment]
Sorry for the repeated posts.  If I disable Camera Dock then lib-wrapper is no longer angry in my test world.  Now to see if the same can be said in my production world.

**Comment by MythicalRollers** [comment]
Okay, can confirm.  A/V QoL and Camera Dock do not like each other.  If they are both active, lib-wrapper is not happy even though Camera Dock doesn't use Lib-wrapper.  Other modules do use it though, I had to delete Camera Dock, log back into the world and set A/V QoL as priority in lib-wrapper settings.  Then I installed Camera Dock again and I was able to get into my world without it freezing.  The Lib-wrapper error does get logged in the console, but the 28000+ messages about rendering camera stopped.
TL:DR A/V QoL and Camera Dock do not like each other.  If you use them and other Lib-wrapper modules you'll get errors and world will freeze.  Only solution is to delete Camera Dock form world, and make A/V a priority in Lib-wrapper.  Only then can you install Camera Dock again and your world not freeze.  The Lib-wrapper still those errors, but at this time I do not see any hiccups in my world from it.

**Comment by patrickporto** [comment]
@MythicalRollers you can use A/v QoL without Lib-wrapper. However I will look for a fix to it

**Comment by patrickporto** [comment]
@MythicalRollers I don't get reproducing the problem. I tried here, so Camera Dock, Lib Wrapper and A/V QoL work together. I believe that it is possible some specific problem happened in the Foundry modules loading, but I don't know where it is.

I will close this issue till the problem to happens again.

**Comment by MythicalRollers** [comment]
I made a new world, and only installed 5 modules.  One of which I disabled.
A/V QoL
Camera Dock
Libwrapper
LiveKit AV
My own compendium (which is not enabled)
See snippet for error.
![Screenshot 2023-02-28 171156](https://user-images.githubusercontent.com/106455111/222004409-11f4498d-737e-4ce4-9feb-d237e0e301e4.png)

**Comment by MythicalRollers** [comment]
Update, Camera Dock has a github already about this, with the resolution of "Can't figure out how to stop this error, ignore it".  Just FYI incase this comes up again.
https://github.com/theripper93/camera-dock/issues/5
